doc/cortx-minikube-k8s-installation.md

```diff
@@ -0,0 +1,258 @@
+# **CORTX on MINIKUBE - Quick Install Guide**
+
+**1. Minimum Requirements:**
+
+* **RAM**: 10GB
+* **Processor**: 6
+* **NIC**: 1
+* **OS Disk**: 1 disk of 20GB
+* **Data Disks**: 4 disks of 10GB each
+* **Metadata disks**: 2 disks of 10GB each
+* **Container or virtual machine manager**, such as: Docker, Hyperkit, Hyper-V, KVM, Parallels, Podman, VirtualBox, or VMware Fusion/Workstation
+
+
+**2. Install Kubectl**
+
+
+```
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+```
+ 
+```
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+```
+
+```
+kubectl version
+```
+
+**3. Install Helm(A package manager for K8s):**
+
+```
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+```
+
+```
+chmod 700 get_helm.sh
+```
+
+```
+./get_helm.sh
+```
+
+**4. Install and start minikube(Centos7.9):**
+
+```
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+```
+
+```
+sudo install minikube-linux-amd64 /usr/local/bin/minikube
+```
+
+```
+minikube start --driver=none
+```
+
+Note minikube may fail to start in some setup due to driver issue, if this happens you can install the drivers here -> Drivers | minikube (k8s.io)
```

Comment:
@alfhad Could you please include the link to minikube [drivers ](https://minikube.sigs.k8s.io/docs/drivers/)